### PR TITLE
Add a mongo support module

### PR DESCRIPTION
**Pull request by derrickburns**
This is used for creating a connection string from individual environment variables. 

One could argue that it should be in its own repo, but who wants to deal with that. :)

On the bright side, I did write unit tests!  They pass.

### CODE REVIEW

**Comment by derrickburns** [comment]
@pazaan MONGO_OPT_PARAMS is of the form 
OPT_PARAMS ::=  name "=" value "&" OPT_PARAMS
OPT_PARAMS ::=  name "=" value

So, multiple pairs of name/value pairs can be provided.

**Comment by pazaan** [comment]
@derrickburns If multiple pairs of name/value pairs can be provided, then there should be a test for that as well 😄

**Comment by derrickburns** [comment]
They are treated as a string. Never parsed. So nothing more to test.

Derrick Burns

File: index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@
 exports.arrays = require('./lib/arrays.js');
 exports.base32hex = require('./lib/base32hex.js');
 exports.config = require('./lib/config.js');
+exports.mongo = require('./lib/mongo.js');
```
Comment (pazaan) [inline index.js]: Should we name this export differently, to disambiguate it from, say, the mongo driver? Perhaps `exports.mongoUtil`?

Comment (derrickburns) [reply index.js]: There are only two hard things in Computer Science: cache invalidation and naming things.

-- the late Phil Karlton

Comment (derrickburns) [reply index.js]: @pazaan Unless you feel strongly, I am inclined to decline this request.  I don't think it warrants cycling through all the clients. :)

**Pull request merged**